lms/templates/instructor/edx_ace/accountcreationandenrollment/email/body.txt

```diff
@@ -5,7 +5,8 @@
 {% blocktrans %}email: {{ email_address }}{% endblocktrans %}
 {% blocktrans %}password: {{ password }}{% endblocktrans %}
 
-
 {% blocktrans %}It is recommended that you change your password.{% endblocktrans %}
 
-{% blocktrans %}Sincerely yours, The {{ course_name }} Team{% endblocktrans %}{% endautoescape %}
\ No newline at end of file
+{% blocktrans %}You may access your course at{% endblocktrans %}: {{ course_url }}.
```

Comment:
Everything, including punctuation should be inside `blocktrans` since some languages may want to position the link at some other place, use something else instead of a colon, etc.
The link should be interpolated into the translated string using a placholder.

Reply:
@mtyaka True, the colon should be inside the translated string, thanks for noticing. It seems that even the whitespace after the colon should be inside (if we follow the example set by the other strings in this e-mail).

However, the link part contains HTML (in the HTML version of the e-mail), and there's the recommendation of not using HTML inside translated strings, as mentioned in the „notes“ section of this PR ([point 3 here](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#general-internationalization-rules)), so I think that text and link can be separated.
Other options:
- use `with`, like [in this example](https://github.com/edx/edx-platform/blob/94c3b4ee5a1406d57aef2929a11448b4deb8476d/lms/templates/wiki/includes/editor_widget.html#L4), to store the link opening and link closing HTML. [Some examples in the translated strings](https://github.com/edx/edx-platform/blob/94c3b4ee5a1406d57aef2929a11448b4deb8476d/conf/locale/en/LC_MESSAGES/django.po#L754). 
- use a `course_url_link` variable which contains the HTML, instead of `course_url` with just the link

It seem the first one (`{link_start}{course_url}{link_end}`) is more flexible and is an improvement.
@mtyaka is that the best option in this case? (If yes, @gr4yscale could you please change it and also include the colon?)

Reply:
@clemente @gr4yscale Sorry, I didn't notice your response until now. It looks like the first option with `link_start` and `link_end` variables is used in several places through the codebase, so I think we should do the same here 👍

Reply:
@mtyaka I've made the proposed changes, thanks for the suggestion, and thanks @clemente for identifying some examples! 

```
edx.devstack.lms     | ------- EMAIL -------
edx.devstack.lms     | To: hithere@testing123.com
edx.devstack.lms     | From: Your Platform Name Here
edx.devstack.lms     | Subject: You have been enrolled in la la la test
edx.devstack.lms     | Body:
edx.devstack.lms     | Welcome to la la la test
edx.devstack.lms     | 
edx.devstack.lms     | To get started, please visit https://localhost:18000. The login information for your account follows.
edx.devstack.lms     | 
edx.devstack.lms     | email: hithere@testing123.com
edx.devstack.lms     | password: PbpHd436NXBF
edx.devstack.lms     | 
edx.devstack.lms     | It is recommended that you change your password.
edx.devstack.lms     | 
edx.devstack.lms     | You may access your course at: http://localhost:18000/courses/course-v1:mine+yeah101+2019_T1/.
edx.devstack.lms     | 
edx.devstack.lms     | Sincerely yours, The la la la test Team
edx.devstack.lms     | -------  END  -------
```